cmd/kubelet/app/server.go

```diff
@@ -199,12 +202,14 @@ func NewKubeletServer() *KubeletServer {
 		RootDirectory:     defaultRootDir,
 		SyncFrequency:     10 * time.Second,
 		SystemContainer:   "",
+		ReconcileCIDR:     true,
 	}
 }
 
 // AddFlags adds flags for a specific KubeletServer to the specified FlagSet
 func (s *KubeletServer) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&s.Config, "config", s.Config, "Path to the config file or directory of files")
+	fs.BoolVar(&s.Accepting, "accepting", s.Accepting, "Accepting condition")
```

Comment:
This flag description should be a bit more fleshed out. What does "accepting condition" mean? When should it be set? You shouldn't have to look through the file change history and find this PR to answer those questions.